clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -7017,6 +7017,7 @@ static void ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D,
   case ParsedAttr::AT_RustAllocator:
     handleSimpleAttribute<RustAllocatorAttr>(S, D, AL);
     break;
+    
```

Comment:
This is probably my fault, but can you remove this newline?

Also, please be sure you're not auto formatting code in your editor, such as removing newlines at the end of files.